dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckerTest.java

```diff
@@ -279,10 +296,65 @@ public void testInheritanceWithGuavaCollectionInheritance() throws URISyntaxExce
             Paths.get(URLClassLoader.getSystemResource("testdata/guava-26.0-jre.jar").toURI()));
     List<FullyQualifiedMethodSignature> methodsNotFound =
         StaticLinkageChecker.findUnresolvedReferences(
-            pathsForJar, Arrays.asList(guavaCollectionPut));
+            pathsForJar, Arrays.asList(guavaCollectionPut), new HashSet<>());
     Truth.assertThat(methodsNotFound).hasSize(0);
   }
 
+  @Test
+  public void testFindUnresolvedReferences_unusedLzmaClassByGrpc()
+      throws RepositoryException, IOException, ClassNotFoundException {
+    String bigTableCoordinate = "com.google.cloud:google-cloud-bigtable:jar:0.66.0-alpha";
+    List<Path> paths = StaticLinkageChecker.coordinateToClasspath(bigTableCoordinate);
+    Truth.assertThat(paths).isNotEmpty();
+
+    // Prior to class usage graph traversal, there was linkage error for lzma-java classes.
+    List<FullyQualifiedMethodSignature> unresolvedMethodReferences =
+        staticLinkageChecker.findUnresolvedMethodReferences(paths);
+
+    Assert.assertThat(
```

Comment:
Truth gives you better error messages. If it's not empty, the message will contain the actual list.

```
Truth.assertWithMessage("Because...").that(unresolvedMethodReferences).isEmpty();
```